index.es6.js

```diff
@@ -51,7 +52,7 @@ export default React.createClass({
       return
     }
 
-    if (!('checked' in this.props)) {
+    if (!('checked' in this.props) || this.state.pendingAllowed) {
```

Comment:
This should probably use `this.props.allowPending` directly, as I'm not seeing any other uses of `this.state.pendingAllowed`